### ISSUE TITLE
Azbrowse usage doesn't show `version` command

### ISSUE THREAD

**Issue by stuartleeks** [lawrencegripper/azbrowse#184]
Output from `azbrowse --help` doesn't include the version command:

```bash
$ azbrowse --help
Loading db ...
Loading db complete
Usage of run:
  -debug
    	run in debug mode
  -demo
    	run in demo mode to filter sensitive output
  -fuzzer int
    	run fuzzer (optionally specify the duration in minutes) (default -1)
  -navigate string
    	navigate to resource
```

**Comment by jjcollinge** [comment]
Will fix this up later today

**Comment by jjcollinge** [comment]
closed by #190
